examples/nginx-deployment.yaml

```diff
@@ -1,17 +1,21 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: nginx-deployment
+  labels:
+    app: nginx
 spec:
   replicas: 3
+  selector:
+    matchLabels:
+      app: nginx
   template:
     metadata:
       labels:
         app: nginx
     spec:
       containers:
       - name: nginx
-        image: nginx:1.7.9
+        image: nginx:1.15.4
```

Comment:
Is there anything wrong with using `latest` so we don't have to keep bumping it in the future? Obviously not good practice but given it's an example I think it should be fine.

Reply:
I'm keeping it on pace with https://k8s.io/examples/controllers/nginx-deployment.yaml. Otherwise I don't have a good reason :/